src/Dapr.PluggableComponents/Adaptors/QueryableStateStoreAdaptor.cs

```diff
@@ -0,0 +1,60 @@
+using Dapr.PluggableComponents.Components.StateStore;
+using Dapr.PluggableComponents.Utilities;
+using Dapr.Proto.Components.V1;
+using Google.Protobuf;
+using Grpc.Core;
+using Microsoft.Extensions.Logging;
+using static Dapr.Proto.Components.V1.QueriableStateStore;
+
+namespace Dapr.PluggableComponents.Adaptors;
+
+/// <remarks>
+/// .NET prefers the "queryable" spelling compared to the protos definition.
+/// </remarks>
+public class QueryableStateStoreAdaptor : QueriableStateStoreBase
```

Comment:
I noticed that you have been using the word `Adapter` for those gRPC classes. It's not a problem, but I just want to make sure that we are consistent across SDKs when it comes to naming. 

In the go sdk we are using the word `Wrapper` [see this](https://github.com/dapr-sandbox/components-go-sdk/blob/main/state/v1/wrapper.go)

I think it's not a problem to change from wrapper to adaptor, you can argue that the `adaptor` is the way to go in the C# world and it will be fine, I'm ok with it.

Reply:
"Wrapper" tends to be a discouraged term for types in .NET; if it weren't a publicly-exposed type I don't think it'd matter.  Whether it *should* be publicly-exposed is certainly a valid point of discussion; I tend to lean toward exposing those types, and the gRPC `.proto` generated types on which they depend, to allow developers the flexibility of overriding our own behavior if/when it doesn't meet their needs.  That said, I'm not completely opposed to making all of that internal.

Adaptor makes sense to me in that it's "adapting" the (protocol-less) abstract implementation of a state store to a specific transmission protocol (i.e. gRPC).  I don't have a super strong opinion about that or any other similar term, however.